lua/terrortown/lang/en.lua

```diff
@@ -2189,5 +2176,15 @@ L.magneto_stick_help_carry_rag_drop = "Drop ragdoll"
 L.magneto_stick_help_carry_prop_release = "Release prop"
 L.magneto_stick_help_carry_prop_drop = "Drop prop"
 
+-- 2024-01-27
+L.decoy_help_primary = "Throw Decoy on the ground"
+L.decoy_help_secondary = "Stick Decoy to surface"
+
+--2024-01-29
+L.marker_vision_visible_for_0 = "Visible for you"
+L.marker_vision_visible_for_1 = "Visible for your role"
+L.marker_vision_visible_for_2 = "Visible for your team"
+L.marker_vision_visible_for_3 = "Visible for everyone"
+
```

Comment:
Oh wait, this shoulnt do any changes in en. Probably something wrong with your rebase?

Reply:
nah, take a look at the dates, they were previously in the wrong order, now they aren't anymore. It is not important, but not a problem either. It is fine with me